provd/proto/accessibility.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package accessibility;
+
+service AccessibilityService {
+    rpc GetHighContract(Empty) returns (GetHighConstrastResponse) {}
+    rpc SetHighContract(SetHighConstrastRequest) returns (Empty) {}
+    rpc GetReducedMotion(Empty) returns (GetReducedMotionResponse) {}
+    rpc SetReducedMotion(SetReducedMotionRequest) returns (Empty) {}
+    rpc GetScreenReader(Empty) returns (GetScreenReaderResponse) {}
+    rpc SetScreenReader(SetScreenReaderRequest) returns (Empty) {}
+    rpc GetLargeText(Empty) returns (GetLargeTextResponse) {}
+    rpc SetLargeText(SetLargeTextRequest) returns (Empty) {}
+}
+
+message Empty {}
```

Comment:
I seem to recall there being an Empty type available at `import "google/protobuf/Empty.proto"`.

Is this used or avoided @didrocks?

Reply:
The real answer is "it depends" :)

More generally, I don’t know on any best practice here, but the rule of thought here is following the Go proverb "a little copying is better than a little dependency" (https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s). If we are only importing from google protobuf "Empty", let’s try to avoid this and copy the empty message type (this is what we do in other projects). This is in the end easier to setup, less risk of missing the dep and so on…

Once we are importing more proto files from this repo, then, we can move to the google’s version, which is binary compatible anyway :)